doc-build/action.yml

```diff
@@ -1,44 +1,86 @@
-name: "Documentation build."
-description: "Build library documentation using Sphinx."
+name: >
+  Documentation build action
+
+description: >
+  Build library documentation using `Sphinx
+  <https://www.sphinx-doc.org/en/master/>`_. The action installs and uses
+  `sphinx-build <https://www.sphinx-doc.org/en/master/man/sphinx-build.html>`_
+  to generate documentation from the source. It requires that all the
+  documentation is contained in the ``doc/`` directory of a project.
+  Documentation dependencies need to be declared in a
+  ``requirements/requirements_doc.txt`` file or in the ``[doc]`` section of the
+  additional dependencies in the ``pyproject.toml`` file. The action locales the
+  ``doc/Makefile`` and runs the ``make html``, ``make pdf``, and ``make json``
+  (this last one only if not avoided).
```

Comment:
```suggestion
  ``doc/Makefile`` directory and runs the ``make html``, ``make pdf``, and
  ``make json`` commands. This last command is run only if not avoided.
```
I don't know what you mean by "only if not avoided". Could it be changed to "only if required" instead?

Reply:
I agree this last sentence was confusing. Just reformatted it to:

```
If desired, the ``make json`` rule can also be executed to generate JSON documentation.
```